.travis.yml

```diff
@@ -10,12 +10,18 @@ before_install:
 install:
 -  npm install
 -  npm install -g grunt-cli
+-  pip install .
 -  pip install -r requirements_dev.txt
 -  pip install coveralls
 script:
+# Test that we can run from source directory
 - bin/kalite start --traceback -v 2
 - bin/kalite status
 - bin/kalite stop --traceback -v 2
+# Test that we can run as the installed script
```

Comment:
If you just want to test the the script is installed (and thus executable from any directory) why not just try one (fast) command? Unless you have reason to believe that it may or may not start depending on how its called.